src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public String getStartMessage() {
+        return "Bulldozer start working";
+    }
+
+    public String getStopMessage() {
+        return "Bulldozer stop working";
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(getStartMessage());

```

Comment:
here and other places: I don't think we gain something by delegating message to a separate method, just print it right here